### PR TITLE
Replace custom parallel vacuum with vacuumdb

### PR DESCRIPTION
**Pull request by benwh**
Also fix a misleadingly-named parameter in usage.

### CODE REVIEW

File: cmd/draupnir-finalise-image [hunk 1]
```diff
@@ -111,7 +111,7 @@ for database in \$(psql -t -c "select datname from pg_database where not datname
 do
   echo ">> vacuuming \$database"
   psql --dbname "\$database" -t -c "select table_name from information_schema.tables where table_schema='public';" \
-    | xargs -P5 -n1 "$VACUUMDB" --dbname "\$database" --table
+    | xargs --no-run-if-empty -P5 -n1 "$VACUUMDB" --dbname "\$database"
```
Comment (lawrencejones) [inline cmd/draupnir-finalise-image]: We can do better than this now that vacuumdb supports the jobs parameter.

Shall we update this PR or close it @benwh?

Comment (benwh) [reply cmd/draupnir-finalise-image]: Good point - done! Ready for another look

File: cmd/draupnir-finalise-image [hunk 2]
```diff
@@ -124,15 +124,7 @@ echo "Executing anonymisation script $ANON_FILE"
...
-sudo -u postgres bash <<BASH
-export PGPORT="$PORT"
-for database in \$(psql -t -c "select datname from pg_database where not datname like 'template%';");
-do
-  echo ">> vacuuming \$database"
-  psql --dbname "\$database" -t -c "select table_name from information_schema.tables where table_schema='public';" \
-    | xargs -P5 -n1 "$VACUUMDB" --dbname "\$database" --table
-done
-BASH
+sudo -u postgres vacuumdb --all --port="$PORT" --jobs="$(nproc)"
```
Comment (lawrencejones) [inline cmd/draupnir-finalise-image]: Ah wait! `$VACUUMDB`

Comment (benwh) [reply cmd/draupnir-finalise-image]: Oops. Not really sure why we do this, as the `vacuumdb` command is provided on $PATH via a helper in Ubuntu (unlike `pg_ctl`), but updated for consistency anyway!

Comment (lawrencejones) [reply cmd/draupnir-finalise-image]: I assume it's for multi-version installs, but given we don't parameterise it with a version it's not helping us so much.

**Pull request merged**